server/controllers/auth.js

```diff
@@ -28,11 +29,16 @@ exports.registerUser = asyncHandler(async (req, res, next) => {
     password,
   });
 
+  const profile = await Profile.create({
+    userId: user._id,
+    firstName: 'John',
```

Comment:
Are we using the username field in the app? If not, we can change it to request the user to enter their first and last name during the register process itself.